composer.json

```diff
@@ -13,22 +13,9 @@
     "apigen/apigen": "^4",
     "nette/utils": "~2.3.0",
     "phpunit/phpunit": "6.*",
-    "woocommerce/woocommerce-git-hooks": "*",
```

Comment:
Should we archive the `woocommerce/woocommerce-git-hooks` repo?

Reply:
No need, people may use it.